### PR TITLE
(maint) Update packaging minimum version

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,6 +13,6 @@ end
 gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.7")
 gem "beaker-abs", *location_for(ENV['BEAKER_ABS_VERSION'] || "~> 0.1")
 gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.15')
-gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99.7')
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99.40')
```
Comment (mwaggett) [inline Gemfile]: latest is .42

Comment (mwaggett) [reply Gemfile]: I don't see a reason not to use that

Comment (lucywyman) [reply Gemfile]: Updated

File: Gemfile [hunk 2]
```diff
@@ -13,6 +13,6 @@ end
 gem "beaker-hostgenerator", *location_for(ENV['BEAKER_HOSTGENERATOR_VERSION'] || "~> 0.7")
 gem "beaker-abs", *location_for(ENV['BEAKER_ABS_VERSION'] || "~> 0.1")
 gem 'vanagon', *location_for(ENV['VANAGON_LOCATION'] || '~> 0.15')
-gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99.7')
+gem 'packaging', *location_for(ENV['PACKAGING_LOCATION'] || '~> 0.99.42')
```
Comment (donoghuc) [inline Gemfile]: Why is this necessary?

Comment (lucywyman) [reply Gemfile]: Jenkins was picking up old version of the gem (I think for several projects), which caused shipping packages to fail. This ensured we actually got the latest version

Comment (donoghuc) [reply Gemfile]: hmm, that seems weird. I would expect the latest version to be pulled in with original. The only reason I could think of it would not is if a gem depends on pinned version of the packaging gem. If that were the case then this would still not work though. 

Could that be what is happening? I'm fine merging this if it is fixing something, but it seems like we should understand what actually caused the issue.

Comment (mwaggett) [reply Gemfile]: @donoghuc We're still working to determine the cause of the issue - this is happening to like every project in CI (but not locally..). I agree that the latest version should be pulled, but the fact these projects actually *need* a new enough version indicates to me that it should be pinned (support for platforms like el8 and redhatfips weren't added until later versions).

Comment (donoghuc) [reply Gemfile]: @mwaggett I was out friday afternoon. Missed much of the context around this. I will merge this, I am interested to hear what you find out for the root cause. Thanks.
